pkg/operation/seed/scheduler/gardener_kube_scheduler_bootstrap.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scheduler
+
+import (
+	"github.com/gardener/gardener/pkg/features"
+	gardenletfeatures "github.com/gardener/gardener/pkg/gardenlet/features"
+	"github.com/gardener/gardener/pkg/operation/botanist/component"
+	"github.com/gardener/gardener/pkg/operation/botanist/seedsystemcomponents/seedadmission"
+	"github.com/gardener/gardener/pkg/operation/seed/scheduler/configurator"
+	schedulerconfigv18 "github.com/gardener/gardener/pkg/operation/seed/scheduler/v18"
+	schedulerconfigv19 "github.com/gardener/gardener/pkg/operation/seed/scheduler/v19"
+	schedulerconfigv20 "github.com/gardener/gardener/pkg/operation/seed/scheduler/v20"
+	seedadmissionpkg "github.com/gardener/gardener/pkg/seedadmission"
+	"github.com/gardener/gardener/pkg/utils/imagevector"
+	schedulerconfigv18v1alpha2 "k8s.io/kube-scheduler/v18/v1alpha2"
+	schedulerconfigv19v1beta1 "k8s.io/kube-scheduler/v19/v1beta1"
+	schedulerconfigv20v1beta1 "k8s.io/kube-scheduler/v20/v1beta1"
```

Comment:
Can you explain why you chose to copy `v1beta1`, as well? IIUC, we could directly refer to `https://github.com/kubernetes/kube-scheduler`, without copying the code.

Reply:
- If the Seed cluster is v1.18 only `v1alpha1` and `v1alpha2` are available https://github.com/kubernetes/kube-scheduler/tree/v0.18.14/config
- If the Seed cluster is v1.19 or v1.20 only `v1beta1` is available https://github.com/kubernetes/kube-scheduler/tree/v0.19.0/config

we cannot simply upgrade the dependency to 1.19 or 1.20, because we would lose the `v1alpha1` and `v1alpha2` API for 1.18